services/debugger/cache_test.go

```diff
@@ -0,0 +1,69 @@
+package debugger
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"time"
+)
+
+var _ = Describe("cache", func() {
+
+	Context("cache testing", func() {
+		testKey := "test_key"
+		testValue := []byte("test_value")
+
+		It("Cache init", func() {
+			var c CacheT
+			c.init()
+			Expect(c.size).NotTo(Equal(0))
+			Expect(c.keyTTL).NotTo(Equal(0))
+			Expect(c.cacheMap).NotTo(BeNil())
+		})
+
+		It("Cache update", func() {
+			var c CacheT
+			c.Update(testKey, testValue)
+			Expect(len(c.cacheMap)).To(Equal(1))
+			Expect(len(c.cacheMap[testKey].data)).To(Equal(1))
+			Expect(c.cacheMap[testKey].data[0]).To(Equal(testValue))
+		})
+
+		It("Cache timeout", func() {
+			var c CacheT
+			c.init()
+			c.keyTTL = time.Second
+			c.cleanupFreq = time.Second
```

Comment:
c.init() should not be called from outside the methods of the structs
It also should not be called before variables are specified

time.Second is to much for a test,
```suggestion
		    c := CacheT{
		        keyTTL:  10 * time.Millisecond, 
     			cleanupFreq: 10 * time.Millisecond,
		    }
```